packages/ses/src/global-object.js

```diff
@@ -1,8 +1,47 @@
-import { defineProperty, objectHasOwnProperty, entries } from './commons.js';
+import {
+  TypeError,
+  assign,
+  create,
+  defineProperty,
+  entries,
+  freeze,
+  objectHasOwnProperty,
+  unscopablesSymbol,
+} from './commons.js';
 import { makeEvalFunction } from './make-eval-function.js';
 import { makeFunctionConstructor } from './make-function-constructor.js';
 import { constantProperties, universalPropertyNames } from './whitelist.js';
 
+/**
+ * The host's ordinary global object is not provided by a with block,
```

Comment:
```suggestion
 * The host's ordinary global object is not provided by a `with` block,
```
and likewise elsewhere. My preference is to do this for all embedded code text, but we should do it at least for code that also looks like a normal standalone english word.